.circleci/config.yml

```diff
@@ -207,6 +207,44 @@ jobs:
                     aws s3 sync --only-show-errors . s3://cardigan.wellcomecollection.org
                 popd
             fi
+  edge_lambdas_build:
+    docker:
+      - image: circleci/node:8.11.3
+    working_directory: /home/circleci/wellcomecollection.org/router/webapp
+    steps:
+      - add_ssh_keys
+      - restore_cache:
+          key: repo_{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Install deps
+          command: yarn
+      - run:
+          name: Run tests
+          command: yarn test
+      - run:
+          name: Build lambda ZIP
+          command: |
+            zip -r edge_lambda_origin.zip ab_testing.js origin.js redirector.js redirects.js wiRedirector.js
+            mkdir -p /tmp/artifacts
+            cp edge_lambda_origin.zip /tmp/artifacts
+            ls /tmp/artifacts
+      - persist_to_workspace:
+          root: /tmp/artifacts
+          paths:
+            - edge_lambda_origin.zip
+  edge_lambdas_deploy:
+    docker:
+      - image: circleci/node:8.11.3
+    working_directory: /tmp
+    steps:
+      - attach_workspace:
+          at: /tmp/artifacts
+      - run:
+          name: Install awscli
```

Comment:
Really need to put this in a container and call it `weco/circle_node:8.11.3` or something.